search/src/test/scala/weco/api/search/works/WorksFiltersTest.scala

```diff
@@ -289,13 +269,18 @@ class WorksFiltersTest
     it("filters by to date") {
       withWorksApi {
         case (worksIndex, routes) =>
-          insertIntoElasticsearch(worksIndex, work1709, work1950, work2000)
+          indexTestDocuments(worksIndex, productionWorks: _*)
+
           assertJsonResponse(
             routes,
             s"$rootPath/works?production.dates.to=1960-01-01"
```

Comment:
Does this one want a keyword arg like 255?

Reply:
Yup, good spot, thanks.